README.md

```diff
@@ -1,46 +1,32 @@
-# Getting Started with Create React App
+# UI Bridge for Substrate chains.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is a UI proposal for querying , operating  and testing the bridge.
 
-## Available Scripts
+## Installation.
 
-In the project directory, you can run:
+### `yarn`
+
+This will install all the dependencies for the project.
 
 ### `yarn start`
 
 Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
-
-### `yarn test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `yarn build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Open [http://localhost:3001](http://localhost:3001) to view it in the browser.
 
-### `yarn eject`
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+### Configure custom substrate providers
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+In the `.env` file at root project directory, you have to define the following env variables:
```

Comment:
Why not have this file as part of the repo configured for Rialto/Millau public nodes? Or at least providing an example file?

Reply:
sure ... i can provide .env.example in the repo